netcon/mtclient/mtgametrack.h

```diff
@@ -96,8 +96,7 @@
 #pragma pack(1)
 
 // Definitions
-#define GAMETRACKER "209.223.64.251" // The scanner machine at Outrage
-#define GAMETRACKERNAME "gt.parallaxonline.com"
+#define GAMETRACKERNAME "tracker.pxo.nottheeye.com"
```

Comment:
`#define GAMETRACKERNAME CHATTRACKERNAME` if it is usually the same

Reply:
There three defines - GAMETRACKER, CHATTRACKER and PLAYERTRACKER. In this case they are identical, but can be different. I'd prefer to keep them separated.